compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -953,6 +955,41 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
         }
     }
 
+    fn suggest_using_iter_mut(&self, err: &mut DiagnosticBuilder<'_, ErrorGuaranteed>) {
+        let source = self.body.source;
+        let hir = self.infcx.tcx.hir();
+        if let InstanceDef::Item(def_id) = source.instance
+            && let Some(Node::Expr( hir::Expr {hir_id, kind, ..})) = hir.get_if_local(def_id)
```

Comment:
```suggestion
            && let Some(Node::Expr(hir::Expr { hir_id, kind, ..})) = hir.get_if_local(def_id)
```
nit: extraneous space and lack of a space